api/src/main/java/io/opencensus/stats/MeasureMap.java

```diff
@@ -19,6 +19,7 @@
 import io.opencensus.stats.Measure.MeasureDouble;
 import io.opencensus.stats.Measure.MeasureLong;
 import io.opencensus.tags.TagContext;
+import io.opencensus.trace.SpanContext;
```

Comment:
I'm concerned about adding a direct dependency on trace from stats (I can easily see wanting to add in the other direction too). Is there any way to accomplish the same thing in a more abstract way?

Reply:
The only solution I can think of is to extract SpanContext (or create an equivalent class) to the common package, so that it can be shared between trace and stats.